kyaml/setters2/settersutil/settercreator.go

```diff
@@ -64,7 +64,7 @@ func (c *SetterCreator) Filter(input []*yaml.RNode) ([]*yaml.RNode, error) {
 	return nil, c.Create()
 }
 
-func (c SetterCreator) Create() error {
+func (c *SetterCreator) Create() error {
```

Comment:
This is because of the attempt to assign at L113: `c.SettersSchema = sc` does nothing on a non-pointer receiver